docs/source/conf.py

```diff
@@ -49,7 +49,7 @@
 ]
 
 
-def generate_apidocs(*args):
+def generate_apidocs(*args) -> None:
```

Comment:
could we keep the original file, it is based on a automatic template from sphinx

Reply:
yes.  It will need to be added to the exclude list.